### DISCUSSION TITLE
Is "Open Build Service" an option?

### DISCUSSION THREAD

**Discussion by Foexle11** [noisetorch/NoiseTorch#389] [Ideas (I would suggest ...)]
Hey there,
I stumbled across the "Open Build Service" while browsing, and since there are a few discussions around distributing using Flatpack or other package managers, I wanted to mention the project here. I have no Idea if it suits Noisetorch's needs, but it may be a possibility to build packages for different distributions without extra work for every package. You can find the OBS here: https://openbuildservice.org/ .

**Comment by Technetium1** [comment]
That's a great option to keep on the table. I think that project is more intended for large programs with longer compile times than the [free tier of Actions allows](https://docs.github.com/en/billing/managing-billing-for-github-actions/about-billing-for-github-actions). For now, it should be reasonable to continue getting platform-agnostic binaries via GitHub's Actions: https://github.com/noisetorch/NoiseTorch/actions.
